.env

```diff
@@ -5,4 +5,4 @@ POSTGRES_USER=zammad
 REDIS_URL=redis://zammad-redis:6379
 RESTART=always
 # don't forget to add the minus before the version
-VERSION=-5.0.3-14
+VERSION=-5.1.0-1
```

Comment:
We can't bump tag VERSION and container VERSION at the same time

Reply:
Reverted.